### DISCUSSION TITLE
<SVG> SMILL deprecated

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#104] [Perguntas e Respostas]
ano passado cheguei a ver animações bem legais com SVG, até o botão de play do youtube é um SVG que ao ser clicado faz uma animação bem suave para um botão de pause, quando um pessoal disse que era uma animação usando SMILL fiquei algum tempo inspecionando o código fonte e achando aquilo super legal, hoje, no codepen, estava olhando alguns "pens" usando [animações com svg smill](https://css-tricks.com/guide-svg-animations-smil/) e me deparei com a seguinte mensagem no console do chrome:
![svgsmillerror](https://cloud.githubusercontent.com/assets/2836002/14150732/cdd9f4aa-f680-11e5-84a8-f006ac0c1963.png)
pesquisando mais vi este post no stack overflow: [Deprecated SMIL SVG animation replaced with CSS or Web animations effects (hover, click)](http://stackoverflow.com/questions/30965580/deprecated-smil-svg-animation-replaced-with-css-or-web-animations-effects-hover)

agora fico me perguntando, como faríamos uma animação mais complexa com [SVG sem SMILL](https://developer.mozilla.org/en-US/docs/Web/SVG/SVG_animation_with_SMIL)?

**Comment by milacarrera** [comment]
Eu tava lendo sobre isso essa semana e acabai esbarrando com esse artigo no CSS Tricks:
https://css-tricks.com/smil-is-dead-long-live-smil-a-guide-to-alternatives-to-smil-features/

**Comment by felipefialho** [comment]
c/c @willianjusten

**Comment by felipefialho** [comment]
As issues nunca devem ser fechadas @leandroruel, haha :p (só se forem repetidas)
